galley/pkg/crd/validation/webhook.go

```diff
@@ -196,7 +197,7 @@ func NewWebhook(p WebhookParameters) (*Webhook, error) {
 		wh.ownerRefs = []v1.OwnerReference{
 			*v1.NewControllerRef(
 				galleyDeployment,
-				extensionsv1beta1.SchemeGroupVersion.WithKind("Deployment"),
+				appsv1.SchemeGroupVersion.WithKind("Deployment"),
```

Comment:
(referencing line 193) - we should also use the appsv1 clientset interface, i.e. `wh.clientset.AppsV1.Deployments(...)`